common/proxy/proxy_factory.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 type ProxyFactory interface {
-	GetProxy(invoker protocol.Invoker, url *common.URL) *Proxy
+	GetProxy(invoker protocol.Invoker, callBack interface{}, url *common.URL) *Proxy
```

Comment:
why change the parameters of GetProxy? Why not add a GetAsyncProxy? @fangyincheng @hxmhlt pls give a suggestion.

Reply:
i have added GetAsyncProxy method in ProxyFactory